### PR TITLE
serve light client data on `prater` by default

### PR DESCRIPTION
**Pull request by etan-status**
Applies a `prater` testnet specific config default to serve light client
data on that network.

### CODE REVIEW

**Comment by etan-status** [comment]
Followup from https://github.com/status-im/infra-nimbus/pull/91

File: beacon_chain/networking/network_metadata.nim [hunk 1]
```diff
@@ -143,14 +153,29 @@ proc loadEth2NetworkMetadata*(path: string, eth1Network = none(Eth1Network)): Et
       else:
         ""
 
+      configDefaults =
+        if path.endsWith("/prater"):
```
Comment (etan-status) [inline beacon_chain/networking/network_metadata.nim]: @tersec : When merging this to `kiln-dev-auth`, please also add `/kiln` here.

**Comment by etan-status** [comment]
- Fix test suite

File: beacon_chain/networking/network_metadata.nim (hunk 1 above)
Comment (tersec) [reply beacon_chain/networking/network_metadata.nim]: Sure

**Review by tersec — APPROVED** [review]

File: beacon_chain/networking/network_metadata.nim (hunk 1 above)
Comment (zah) [inline beacon_chain/networking/network_metadata.nim]: Er, this is not a good way to check which network you are connected to. The best way is to check the hash of the genesis block in the selected data dir. I've used this in a work-in-progress PR that I can submit today.

**Comment by tersec** [comment]
A general note/caveat: while there are a number of locally-unprotected `get`s here, the BN-based codepaths look okay, and while there are design alternatives to this approach, they have their own tradeoffs.

File: beacon_chain/networking/network_metadata.nim (hunk 1 above)
Comment (etan-status) [reply beacon_chain/networking/network_metadata.nim]: Cool, if you have a better way to detect, let's use that one!

**Comment by etan-status** [comment]
- Use first 40 bytes of `genesisData` (`genesis_time` and `genesis_validators_root`) to identify network instead of `--network` folder name.

File: beacon_chain/networking/network_metadata.nim (hunk 1 above)
Comment (tersec) [reply beacon_chain/networking/network_metadata.nim]: https://github.com/status-im/nimbus-eth2/pull/3494/commits/e88bc1b3500e921f9ddd7e66ce156dc13c6a1d6b

**Pull request merged**